re.rizin.cutter.yml

```diff
@@ -0,0 +1,43 @@
+app-id: re.rizin.cutter
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: cutter
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+
+modules:
+  - name: rizin
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+      - --prefix=/app
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/rizin/releases/download/v0.2.0/rizin-src-v0.2.0.tar.xz
+        sha256: 0861dea36e3bb3285afab56d880433d1790460af69e031d39cb2b616104ebc4f
+
+  - name: cutter
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCUTTER_USE_BUNDLED_RIZIN=off
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCUTTER_ENABLE_KSYNTAXHIGHLIGHTING=on
+      - -DCUTTER_ENABLE_PYTHON=on
+      - -DCUTTER_EXTRA_PLUGIN_DIRS=/app/share/rizin/cutter/plugins\;
```

Comment:
Cutter uses QStandardPaths for calculating the folder of plugins and other resources. I'm not sure if Flatpak version of Qt libraries are patched to use /app/share prefix there. If it is `CUTTER_EXTRA_PLUGIN_DIRS` shouldn't be necessary, if it isn't it needs to be handled in Cutter for more than just plugins.

Reply:
Thanks for the review, I added it while trying to find why the rz-ghidra extension was not loading but it is actually useless : 

 ```
➜  flathub.re.rizin.cutter git:(com.github.rizinorg.Cutter) ✗ flatpak run re.rizin.cutter                                                            
Gtk-Message: 22:14:06.859: Failed to load module "canberra-gtk-module"
Qt: Session management error: Could not open network socket
Icon theme "elementary" not found.
Icon theme "gnome" not found.
"0.2.0" "0.2.0"
Plugins are loaded from "/home/opendev/.var/app/re.rizin.cutter/data/rizin/cutter/plugins"
Loaded 0 plugin(s).
Plugins are loaded from "/app/share/rizin/cutter/plugins"
Plugins are loaded from "/usr/share/rizin/cutter/plugins"
Plugins are loaded from "/usr/share/runtime/share/rizin/cutter/plugins"
Plugins are loaded from "/run/host/share/rizin/cutter/plugins"
```

I'm removing it

Reply:
Not sure why do you think it's useless, unless the line below comes from a build without `-DCUTTER_EXTRA_PLUGIN_DIRS` set.

```
Plugins are loaded from "/app/share/rizin/cutter/plugins"
```

Reply:
@barthalion EXTRA_PLUGIN_DIRS is what it says extra directories for searching plugins the primary source of plugin directories is QStandardPaths . There are two possibilities:
* flatpak Qt runtime is configured to use `/app/share/` in `QStandardPaths` -> there is no need to specify it manually using EXTRA_PLUGIN_DIRS
* flatpak Qt runtime isn't configured to use `/app/share/` in `QStandardPaths` -> there are other files beside plugins that Cutter searches using `QStandardPaths` so setting `EXTRA_PLUGIN_DIRS` is insufficient and it needs to be handled differently. 

It needs to be confirmed which of the 2 cases it is, but in both cases EXTRA_PLUGIN_DIRS is wrong solution.

Reply:
The EXTRA_PLUGIN_DIRS is needed as flatpak extensions can only be added in a custom folder (see latest version)